model/CSVDialog.cc

```diff
@@ -32,8 +32,8 @@
 #include <boost/regex.hpp>
 #include <boost/filesystem.hpp>
    
-#include "certify/extensions.hpp"         
-#include "certify/https_verification.hpp" 
+#include "certify/include/boost/certify/extensions.hpp"         
+#include "certify/include/boost/certify/https_verification.hpp" 
```

Comment:
I would have left this header include as is, and added  certify/boost to the include path, but this will work as well (CERTIFY_HOME stuff above is doing nothing in that case).